README.md

```diff
@@ -656,6 +657,7 @@ BEARER_TOKEN: A valid bearer token for the account to associate the payment with
 | Field                    | required | Description                               |
 | ------------------------ |:--------:| ----------------------------------------- |
 | `amount`                 | Yes      | Amount to refund in pence                 |
+| `refund_amount_available`| No       | Total amount still available for refunds  |
```

Comment:
is this the amount which is available before the refund has been performed or after? Would be helpful to make that explicit in the docs.